src/Opaleye/Internal/Print.hs

```diff
@@ -73,6 +75,14 @@ ppSelectLabel l = text "/*" <+> text (defuseComments (Sql.lLabel l)) <+> text "*
                    . ST.replace (ST.pack "*/") (ST.pack " * / ")
                    . ST.pack
 
+ppSelectAntijoin :: Antijoin -> Doc
+ppSelectAntijoin v =
+  text "SELECT *"
+  $$ text "FROM"
+  $$ ppTable (tableAlias 1 (Sql.antijoinTable v))
```

Comment:
No, `SELECT * FROM x WHERE NOT EXISTS (SELECT * FROM y)` is just fine. This is just lazy copy and pasting to explore a proof of concept :)